pages/linux/ip-rule.md

```diff
@@ -0,0 +1,36 @@
+# ip rule
+
+> IP Routing policy database management.
+> More information: <https://man7.org/linux/man-pages/man8/ip-rule.8.html>.
+
+- Display the routing policy:
+
+`ip rule {{show|list}}`
+
+- Insert a new rule use the source prefix:
+
+`sudo ip rule add from {{192.168.178.2/32}}`
+
+- Insert a new rule use the destination prefix:
+
+`sudo ip rule add to {{192.168.178.2/32}}`
+
+- Delete a new rule use the source prefix:
+
+`sudo ip rule delete from {{192.168.178.2/32}}`
+
+- Delete a new rule use the destination prefix:
+
+`sudo ip rule delete to {{192.168.178.2/32}}`
+
+- Dump all the deleted rules:
```

Comment:
I think `dump` sounds more like `print to the terminal`, even though it is correct. How about just
```suggestion
- Flush all the deleted rules:
```

Reply:
Okay, I'll fix it